### DISCUSSION TITLE
Discord API Extension

### DISCUSSION THREAD

**Discussion by LilyMakesThings** [TurboWarp/extensions#539] [Show and tell]
I'm currently working on a Discord API extension that can interact with webhooks and create embeds. The idea is to try and "scratch-ify" the process of sending webhooks and creating embeds in order to make it more accessible.

Thought I'd share it with you! :)
![image](https://github.com/TurboWarp/extensions/assets/127533508/2a8a7c22-b9a5-487c-bb6f-e0af45d6a7a1)
![image](https://github.com/TurboWarp/extensions/assets/127533508/860eada9-1223-4bfa-a449-6c411f6027eb)

![image](https://github.com/TurboWarp/extensions/assets/127533508/fddec83e-a0d9-4b28-97db-4a80522bf584)
![image](https://github.com/TurboWarp/extensions/assets/127533508/a4202960-75dc-4df0-9f29-2e499f4a7658)
![image](https://github.com/TurboWarp/extensions/assets/127533508/f4318803-6ad9-4d1c-9163-e09987a636bb)
![image](https://github.com/TurboWarp/extensions/assets/127533508/a4bbe173-aa1f-4b51-941b-2f18981b940a)
![image](https://github.com/TurboWarp/extensions/assets/127533508/0f2c2e74-5d5f-497c-818a-b60d5f9ea9c7)
![image](https://github.com/TurboWarp/extensions/assets/127533508/dc0d8159-f32e-497c-ac5d-deb3b36a449a)

**Comment by nico-tome** [comment]
what is the link ?

**Reply by LilyMakesThings** [reply]
The Pull Request for it is here! :) https://github.com/TurboWarp/extensions/pull/553

**Reply by nico-tome** [reply]
yes but how i can use it on turbowarp ?

**Reply by LilyMakesThings** [reply]
wait for it to be merged or paste the code from it

**Reply by nico-tome** [reply]
when i past the code nothing append

**Reply by XeroName** [reply]
try to load it with unsandboxing may solve your problem.

**Reply by nico-tome** [reply]
yeah i have seen last day ^^
